### ISSUE TITLE
Request for Button to link to a new window

### ISSUE THREAD

**Issue by janejacksoncareers** [Simplero/Roadmap#318]
Hi Calvin and your team

I notice that when I use your Button feature, when you click on the button it opens in the same window.  Would it be possible to have the OPTION for the button to open in a NEW window please?

I am using the button to link to my new privacy policy to comply with GDPR and it currently opens in the same window but I'd rather have it open in a new window so it's easy for my readers to click get to my original page after viewing the privacy policy.  Many thanks.

Jane

**Comment by Ryan-Henderson** [comment]
I would use this feature as well Thanks!

**Comment by desireeeast** [comment]
I'd love this too, thanks!

**Comment by kevinltaylor** [comment]
Yes! This seems like such a simple feature that should already be available. A simple checkbox on the button settings to open in new tab/window would be good enough.

We have a page where we link to other resources, but each link takes them away from our site. Having the links open in a new tab would be helpful so they could easily go back to the page on our site to click on other links.
